inc/features/blocks/experience/register.php

```diff
@@ -0,0 +1,102 @@
+<?php
+/**
+ * Experience Block Server Side.
+ *
+ * @phpcs:ignoreFile HM.Files.FunctionFileName.WrongFile
+ *
+ * @package altis-experiments
+ */
+
+namespace Altis\Experiments\Features\Blocks\Experience;
+
+use const Altis\Experiments\ROOT_DIR;
+use function Altis\Experiments\Features\Blocks\get_block_settings;
+use function Altis\Experiments\Utils\get_asset_url;
```

Comment:
`use` should only be used with namespaces: https://github.com/humanmade/coding-standards/issues/158